tests/platform_tests/reboot_timing_constants.py

```diff
@@ -30,7 +30,7 @@
     "COMMON": {
         "PORT_INIT|Start": re.compile(r'.*NOTICE swss#orchagent.*initPort: Initialized port.*'),
         "PORT_READY|Start": re.compile
-        (r'.*swss#orchagent.*updatePortOperStatus.*Port Eth.*oper state set down to up.*'),
+        (r'.*swss#orchagent.*updatePortOperStatus.*Port Eth.*oper state set (down|unknown|up) to up.*'),
```

Comment:
I remember `up to up` was creating issues earlier when the test concluded that port is brought up more than expected times, concluding there was a flap.

Is there a specific reason that you added `up to up` again?

Reply:
In the S6100 upgrade, for physical ports, I only see `up to up` and `down to down`; I don't see `down to up` or `unknown to up` for physical port (I do see the latter for port channels).

Just to check, these messages that should be matched are at/after apply view occurs, right?